akka-docs/src/main/paradox/stream/operators/RetryFlow/withBackoffAndContext.md

```diff
@@ -0,0 +1,54 @@
+# RetryFlow.withBackoffAndContext
+
+Wrap the given @apidoc[FlowWithContext] and retry individual elements in that stream with an exponential backoff. A decider function tests every emitted element and can return a new element to be sent to the wrapped flow for another try.
+
+@ref[Error handling](../index.md#error-handling)
+
+## Signature
+
+Scala
+:   @@signature [RetryFlow.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/RetryFlow.scala) { #withBackoffAndContext }
+
+Java
+:   @@snip [RetryFlowTest.java](/akka-stream-tests/src/test/java/akka/stream/javadsl/RetryFlowTest.java) { #signature }
+
+
+## Description
+
+When an element is emitted by the wrapped `flow` it is passed to the `decideRetry` function, which may return an element to retry in the `flow`. 
+
+The retry backoff is controlled by the `minBackoff`, `maxBackoff` and `randomFactor` parameters.
+At most `maxRetries` will be made after the initial try.
+
+The wrapped `flow` must have **one-in one-out semantics**. It may not filter, nor duplicate elements.
```

Comment:
Make it clear that it will only let a single element in the inner loop at any given time also since that is not how streams in general work.